src/fileAttachment.mjs

```diff
@@ -10,11 +10,38 @@ async function remote_fetch(file) {
   return response;
 }
 
-async function dsv(file, delimiter, {array = false, typed = false} = {}) {
+function asTyper(type) {
+  if (typeof type === "function") {
+    return type;
+  }
+  switch (`${type}`) {
+    case "number": return Number;
+    case "string": return String;
+    case "boolean": return Boolean;
```

Comment:
this differs from autotype in that it will not turn "false" into `false`. is that ok?

Reply:
Ah, no, that wasn’t intentional.